### PR TITLE
Human/Sarathi Resprites

### PR DESCRIPTION
**Pull request by rye-rice**
## About The Pull Request

Ports https://github.com/tgstation/tgstation/pull/66321

Resprites Humans and Sarathi

The stars here are @zestybastard (any%) who did the humans and @Ebin-Halcyon who did the sarathi.

Eyes are 2x1 which is very experimental

## Why It's Good For The Game
Screenshot_3629

Screenshot_3630

## Changelog
:cl: Any%, Halcyon
add: Emote overlays for blushing and snoring
imageadd: Humans have beenn resprited and are now NB
imageadd: Lizards have beenn resprited.
/:cl:

### CODE REVIEW

**Comment by Ebin-Halcyon** [comment]
IT'S.
HERE.

**Comment by tiramisuapimancer** [comment]
speedmarg

**Comment by mel-byond** [comment]
The lack of a nose is a bit strange on the frontview for the human sprites, otherwise I can't point it out what repulses me of that frontview.

The rest looks great!!

**Comment by rye-rice** [comment]
> The lack of a nose is a bit strange on the frontview for the human sprites, otherwise I can't point it out what repulses me of that frontview.
> 
> The rest looks great!!

It looks odd bald. I asked any to experiment with the face of the human, but shes busy working on walls at the moment

**Review by triplezeta — APPROVED** [review]
yea

**Comment by triplezeta** [comment]
m

**Comment by Ebin-Halcyon** [comment]
Stuck in limbo hell.

**Comment by rye-rice** [comment]
Conflicts fixed.

I also added the mouth pixels, which I have been dreading adding but, it seems to be the only way to get the face to not look flat unfortunately.

**Comment by rye-rice** [comment]
Undrafted

**Comment by rye-rice** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request

**Comment by Ebin-Halcyon** [comment]
> This pull request has conflicts, please resolve those before we can evaluate the pull request

You fucker you made me look.

**Comment by rye-rice** [comment]
Tested the mouth pixels, I think it looks fine, the issue now is testing the eyes

File: code/modules/surgery/bodyparts/species_parts/lizard_bodyparts.dm [hunk 1]
```diff
@@ -8,7 +8,7 @@
 	icon = 'icons/mob/species/lizard/bodyparts.dmi'
 	uses_mutcolor = TRUE
 	limb_id = SPECIES_LIZARD
-	is_dimorphic = TRUE //any made enby humans, while halcy made them gendered, making them todays biggest loser
+	is_dimorphic = TRUE
```
Comment (triplezeta) [inline code/modules/surgery/bodyparts/species_parts/lizard_bodyparts.dm]:
```suggestion
```

you don't need to set the value of a var to `TRUE` when it is already set to `TRUE` on the parent object

**Review by triplezeta — APPROVED** [review]

**Pull request merged**